packages/adapter-next/src/index.ts

```diff
@@ -26,16 +27,16 @@ export default defineSliceMachinePlugin<PluginOptions>({
 		hook("slice:update", sliceUpdate);
 		hook("slice:delete", sliceDelete);
 		hook("slice:read", sliceRead);
+		hook("slice:library:read", sliceLibraryRead);
 
 		hook("custom-type:create", customTypeCreate);
 		hook("custom-type:update", customTypeCreate);
 		hook("custom-type:delete", customTypeDelete);
 		hook("custom-type:read", customTypeRead);
+		hook("custom-type:library:read", customTypeLibraryRead);
```

Comment:
Same as `slice-library:read`, what do you think of renaming this to `custom-type-library:read`?

Reply:
LGTM